From 87511a915c86ffc68536f03968b70deba82012b8 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Thu, 21 Dec 2023 15:22:26 -0500
Subject: [PATCH 1/2] =?UTF-8?q?Drop=20AppData=20=E2=80=98content=5Frating?=
 =?UTF-8?q?=E2=80=99=20to=20fix=20#26?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Validation with appstreamcli reports that this element must not have
text content, which in this case was just ‘none’ anyway.
---
 data/net.sourceforge.wavbreaker.appdata.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/net.sourceforge.wavbreaker.appdata.xml b/data/net.sourceforge.wavbreaker.appdata.xml
index 13cabd7..9cfcfde 100644
--- a/data/net.sourceforge.wavbreaker.appdata.xml
+++ b/data/net.sourceforge.wavbreaker.appdata.xml
@@ -3,7 +3,6 @@
 <component type="desktop-application">
   <id>net.sourceforge.wavbreaker</id>
   <metadata_license>FSFAP</metadata_license>
-  <content_rating>none</content_rating>
   <project_license>GPL-2.0+</project_license>
   <name>wavbreaker</name>
   <summary>GUI tool to split WAV, MP2 and MP3 files</summary>

From 2c259c77f664285ead17b5df27f74a562efc2c14 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Tue, 2 Jan 2024 19:37:48 -0500
Subject: [PATCH 2/2] Supply a valid content rating (with  no content
 attributes)

---
 data/net.sourceforge.wavbreaker.appdata.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/net.sourceforge.wavbreaker.appdata.xml b/data/net.sourceforge.wavbreaker.appdata.xml
index 9cfcfde..478463d 100644
--- a/data/net.sourceforge.wavbreaker.appdata.xml
+++ b/data/net.sourceforge.wavbreaker.appdata.xml
@@ -3,6 +3,7 @@
 <component type="desktop-application">
   <id>net.sourceforge.wavbreaker</id>
   <metadata_license>FSFAP</metadata_license>
+  <content_rating type="oars-1.1" />
   <project_license>GPL-2.0+</project_license>
   <name>wavbreaker</name>
   <summary>GUI tool to split WAV, MP2 and MP3 files</summary>
